docs/developer/watcher-local-test.md

```diff
@@ -0,0 +1,127 @@
+# Test watcher locally in two-cluster-setup using K3D
+
+### Contents
+
+### Export customizable attributes
+```shell
+export KLM_PATH=<local-path-of-kyma-lifecycle-manager-project>
+export KLM_VERSION=<docker-image-tag-for-local-testing>
+export KUBE_CONFIG_DIR=<path-to-export-local-kubeconfigs-to>
+```
+
+### Create KCP cluster
+Run the following command to create a local control-plane (KCP) cluster:
+```shell
+k3d cluster create kcp-local --port 9080:80@loadbalancer \
+--registry-create k3d-registry.localhost:0.0.0.0:5111 \
+--k3s-arg '--no-deploy=traefik@server:0'
+```
+
+### Install Istio and Manifest CRD
+Run the following command to install Istio and Manifest CRD on the KCP cluster:
+```shell
+istioctl install -y && \
+kubectl apply -f https://raw.githubusercontent.com/kyma-project/module-manager/main/config/crd/bases/operator.kyma-project.io_manifests.yaml
+```
+
+### Build and push lifecycle-manager image
+1. Run the following command to get KCP cluster kubeconfig and export it:
+```shell
+k3d kubeconfig get kcp-local > $KUBE_CONFIG_DIR/kcp-local.yaml && \
+export KUBECONFIG=$KUBE_CONFIG_DIR/kcp-local.yaml
+```
+2. Run the following command to build and push lifecycle-manager image to local k3d registry:
+```shell
+export K3D_REG=k3d-registry.localhost:5111 && \
+make -C $KLM_PATH docker-build IMG=$K3D_REG/lifecycle-manager:$KLM_VERSION && \
+make -C $KLM_PATH docker-push IMG=$K3D_REG/lifecycle-manager:$KLM_VERSION
+```
+
+### Deploy lifecycle-manager on KCP:
+Run the following command to deploy lifecycle-manager on the KCP cluster:
+```shell
+export K3D_REG=k3d-registry.localhost:5111 && \
+make -C $KLM_PATH install && \
+make -C $KLM_PATH local-deploy-with-watcher IMG=$K3D_REG/lifecycle-manager:$KLM_VERSION
+```
+
+### Apply sample module templates for sample-kyma:
+Run the following command to deploy lifecycle-manager on the KCP cluster:
+```shell
+export KUBECONFIG=$KUBE_CONFIG_DIR/kcp-local.yaml && \
+kubectl apply -f $KLM_PATH/config/samples/component-integration-installed/operator_v1alpha1_moduletemplate_kcp-module.yaml && \
+kubectl apply -f $KLM_PATH/config/samples/component-integration-installed/operator_v1alpha1_moduletemplate_skr-module.yaml
```

Comment:
why do you install `skr-module`. As far as I see below only `kcp-module `is used?